racket-edit.el

```diff
@@ -200,14 +200,22 @@ Please keep in mind the following limitations:
   (interactive "P")
   (pcase (racket--symbol-at-point-or-prompt prefix "Visit definition of: ")
     (`nil nil)
-    (str (if (and (eq major-mode 'racket-mode)
-                  (not (equal (racket--repl-file-name+md5)
-                              (cons (racket--buffer-file-name t) (md5 (current-buffer)))))
-                  (y-or-n-p "Run current buffer first? "))
-             (racket--repl-run nil nil
-                               (lambda (_n/a)
-                                 (racket--do-visit-def-or-mod 'def str)))
-           (racket--do-visit-def-or-mod 'def str)))))
+    (str (racket--visit-symbol-definition str))))
+
+(defsubst racket-lispy-visit-symbol-definition (str)
```

Comment:
Should this be a plain old `defun` instead? Genuine question; I'm not sure. (Inlining across packages seems like it could be trouble. e.g. Someday we need to change this. We do. User updates racket-mode package. But. That doesn't mean lispy package necessarily gets recompiled then. Ergo confusion.)